WORKSPACE

```diff
@@ -35,16 +42,24 @@ bind(
     actual = "//util/python:python_headers",
 )
 
+load("@rules_jvm_external//:repositories.bzl", "rules_jvm_external_deps")
```

Comment:
I'm not sure this is needed if we aren't changing the list of standard maven sources, which this addition doesn't appear to be doing. Did you read otherwise somewhere?

Reply:
Removed this dependency since we won't be using the publish command.